jet-fuel/app/controllers/urls_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UrlsController < ApplicationController
+  def create
+    url = Url.new(url_params)
+    if url.save
+      redirect_to url_path(url)
+    else
+      redirect_to root_path
+    end
+  end
+
+  def show
+    require 'pry'; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.